packages/ethersjs-provider/src/EthersWeb3Provider.ts

```diff
@@ -0,0 +1,61 @@
+import { NetworkId } from "@augurproject/artifacts";
```

Comment:
FYI, you should be able to use the existing EthersProvider instead once https://github.com/AugurProject/augur/pull/1296 gets merged.

Reply:
Right! I'll do another PR to remove these after or maybe handle it in the merge if there's a conflict.